frontend/src/scenes/persons/PersonProperty.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useMemo } from 'react'
+import { BulbOutlined, NumberOutlined } from '@ant-design/icons'
+import { IconText } from 'lib/components/icons'
+import { PropertyKeyInfo } from 'lib/components/PropertyKeyInfo'
+import { Tooltip } from 'antd'
+
+export function PersonProperty({
+    name,
+    value,
+}: {
+    name: string
+    value: string | number | boolean | null
+}): JSX.Element {
+    const type = useMemo(() => {
```

Comment:
This seems a bit suss - don't we have that logic already in frontend/src/lib/components/PropertiesTable.tsx?

Reply:
I was thinking of introducing instead, but it might not make sense right now. I've kept the logic from PropertiesTable, and in a separate PR I'll update the UI (feature flagged) to the new layout.